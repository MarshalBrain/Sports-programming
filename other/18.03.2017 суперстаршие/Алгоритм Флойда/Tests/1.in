5
0 48 18 14 84 
65 0 4 58 5 
76 11 0 95 60 
31 54 88 0 59 
14 60 25 15 0 
