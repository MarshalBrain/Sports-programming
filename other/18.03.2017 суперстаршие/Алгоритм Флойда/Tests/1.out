0 29 18 14 34 
19 0 4 20 5 
30 11 0 31 16 
31 54 49 0 59 
14 36 25 15 0 
